packages/Form/Input/checkbox/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axa-fr/react-toolkit-form-input-checkbox",
-  "version": "1.3.4",
+  "version": "1.3.5",
```

Comment:
You do not need to touch the version manualy. You can let "version": "1.3.4".
Publish action will update it automaticaly.